kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/BigQueryWriter.java

```diff
@@ -98,7 +98,7 @@ protected InsertAllRequest createInsertAllRequest(PartitionedTableId tableId,
   public void writeRows(PartitionedTableId table,
                         SortedMap<SinkRecord, InsertAllRequest.RowToInsert> rows)
       throws BigQueryConnectException, BigQueryException, InterruptedException {
-    logger.debug("writing {} row{} to table {}", rows.size(), rows.size() != 1 ? "s" : "", table);
+    logger.info("writing {} row{} to table {}", rows.size(), rows.size() != 1 ? "s" : "", table);
```

Comment:
This should stay at `DEBUG` level in order to avoid flooding logs with this message when the connector is running healthily. If you need to see this message for debugging purposes, you can either adjust your worker's log4j configuration file or, if you'd rather not restart your worker, you can dynamically adjust log levels using the API introduced in [KIP-495](https://cwiki.apache.org/confluence/display/KAFKA/KIP-495%3A+Dynamically+Adjust+Log+Levels+in+Connect) (more info can be found [here](https://docs.confluent.io/platform/current/connect/logging.html#using-the-kconnect-api) and [here](https://rmoff.net/2020/01/16/changing-the-logging-level-for-kafka-connect-dynamically/)).

Reply:
OK! That makes sense! Let's leave it as debug level then.